src/data/privacy_de.json

```
@@ -45,7 +46,7 @@
                 "<ul><li><a href='/assets/documents/cwa-privacy-notice-1.5-de.pdf'>Datenschutzerklärung Version 1.5 bis 1.7.1 in Deutsch</a></li><li><a href='/assets/documents/cwa-privacy-notice-1.5-en.pdf'>Datenschutzerklärung Version 1.5 bis 1.7.1 in English</a></li><li><a href='/assets/documents/cwa-privacy-notice-1.5-tr.pdf'>Datenschutzerklärung Version 1.5 bis 1.7.1 in Türkisch</a></li></ul>",
                 "<ul><li><a href='/assets/documents/cwa-privacy-notice-1.3-de.pdf'>Datenschutzerklärung bis Version 1.3 in Deutsch</a></li><li><a href='/assets/documents/cwa-privacy-notice-1.3-en.pdf'>Datenschutzerklärung bis Version 1.3 in English</a></li></ul>"
             ],
-            "textcaption": "Letzte Aktualisierung: 12. Juli 2021"
+            "textcaption": "Letzte Aktualisierung: 18. Juli 2021"
```

Shouldn't this be the same date as in the English text i.e. "28. Juli 2021" not "18."?

---

```suggestion
            "textcaption": "Letzte Aktualisierung: 28. Juli 2021"
```
@dsarkar 
In case this is what you need!

---

Indeed, a typo! Thanks!